plugins/evernote.koplugin/main.lua

```diff
@@ -1,22 +1,23 @@
 local BD = require("ui/bidi")
-local InputContainer = require("ui/widget/container/inputcontainer")
-local LoginDialog = require("ui/widget/logindialog")
-local InfoMessage = require("ui/widget/infomessage")
-local NetworkMgr = require("ui/network/manager")
+local DEBUG = require("dbg")
```

Comment:
I said you didn't have to change these, but that DEBUG isn't really the current style. ;-)
```suggestion
local dbg = require("dbg")
```

Reply:
I just figured it might help.